test/index.js

```diff
@@ -152,6 +153,26 @@ describe('KISSmetrics', function () {
       result['Billing Amount'].should.eql(19.95);
       result._n.should.eql('Baked a cake');
     });
+
+    it('should use a POST request if the payload exceeds a threshold', function(done){
+      var largeEvent = helpers.repeat('x', 2100);
+      var timestamp = Math.floor(Date.now() / 1000);
+
+      test
+        .set(settings)
+        .track({event: largeEvent, time: timestamp})
```

Comment:
Hmm @f2prateek @chriso is there a more realistic test for this? Most people don't send a very large event name -- usually it would be because they attached too many properties or traits. 

Also nit pick on line 163-- can we use the fixture convention like you see in the other unit tests? 

@chriso can we also get a unit test to make sure it does not use a POST request if the payload does not exceed the threshold?